roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1674,11 +1674,22 @@ def set_builddefaults_facts(facts):
             builddefaults['git_no_proxy'] = builddefaults['no_proxy']
         # If we're actually defining a builddefaults config then create admission_plugin_config
         # then merge builddefaults[config] structure into admission_plugin_config
+
+        # 'config' is the 'openshift_builddefaults_json' inventory variable
         if 'config' in builddefaults:
             if 'admission_plugin_config' not in facts['master']:
-                facts['master']['admission_plugin_config'] = dict()
+                # Scaffold out the full expected datastructure
+                facts['master']['admission_plugin_config'] = {
```

Comment:
This seems a little redundant with line 1692.

I liked the previous patch set better.

Reply:
I'd rather do this than add more and more conditional statements. Each requiring *yet another description* of why it's even there.